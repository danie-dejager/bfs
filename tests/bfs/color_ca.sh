test "$UNAME" = "Linux" || skip
invoke_bfs . -quit -capable || skip

cd "$TEST"

"$XTOUCH" normal capable
bfs_sudo setcap all+ep capable || skip
ln -s capable link

LS_COLORS="ca=30;41:" bfs_diff . -color
