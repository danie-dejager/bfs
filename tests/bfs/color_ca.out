[01;34m.[0m
[01;34m./[0m[01;36mlink[0m
[01;34m./[0m[30;41mcapable[0m
[01;34m./[0mnormal
